kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.ssl.SslManagerService.xml

```diff
@@ -52,6 +52,34 @@
             required="false"
             default=""
             description="Comma-separated list of allosed ciphers. If not specifed, all Java VM ciphers will be allowed."/>
+        
+         <AD id="ssl.revocation.check.enabled"
+            name="Revocation Check Enabled"
+            type="Boolean"
+            cardinality="0"
+            required="false"
+            default="false"
+            description="Specifies whether to check revocation status of server certificates."/>
```

Comment:
Maybe also here we need to enrich the description of the feature. Also unrelated, but I have noticed that there is an error here: "Comma-separated list of allo**w**ed ciphers"